AeroGearOAuth2/OAuth2Module.swift

```diff
@@ -80,7 +81,9 @@ open class OAuth2Module: AuthzModule {
     open var serverCode: String?
 
     open var idToken: String? { return oauth2Session.idToken }
-    
+
+    private let requestIdentifier: String = ProcessInfo.processInfo.globallyUniqueString
```

Comment:
nice 👍 
I know it's a quick fix but I would think about making it open optional or making it lazy with default value of `ProcessInfo.processInfo.globallyUniqueString` but still being able to overwrite it.